clar2wasm/src/standard/standard.wat

```diff
@@ -640,6 +641,128 @@
         (i64.const 0)
     )
 
+	(func $bit-and (type 1) (param $a_lo i64) (param $a_hi i64) (param $b_lo i64) (param $b_hi i64) (result i64 i64)
+        (local $and_lo i64)
+        (local $and_hi i64)
```

Comment:
Can we remove these local variables, and just leave the results of the `i64.and`s on the stack?

Reply:
Yes i'll get to it

Reply:
done